test/L2VoteAggregator.t.sol

```diff
@@ -384,7 +389,8 @@ contract CastVoteWithReason is L2VoteAggregatorTest {
 
     l2Erc20.mint(address(this), _amount);
 
-    L2GovernorMetadata.Proposal memory l2Proposal = voteAggregator.createProposal(1, 3000, false);
+    L2GovernorMetadata.Proposal memory l2Proposal =
+      voteAggregator.createProposal(1, uint128(voteAggregator.CAST_VOTE_WINDOW()) + 1, false);
```

Comment:
nit: these tests all extend from `L2VoteAggregatorTest`. Can we define a variable there called `DEFAULT_VOTE_END` and set it to `uint128(voteAggregator.CAST_VOTE_WINDOW()) + 1` and use it in all these places?

```suggestion
      voteAggregator.createProposal(1, DEFAULT_VOTE_END, false);
```